Makefile

```diff
@@ -27,7 +27,9 @@ $(MGEN): ./msgp/defs_test.go
 	go generate ./msgp
 
 test: all
+	# keep in sync with 'make travis'
 	go test -v ./msgp
```

Comment:
This and the `travis` rule can probably just be `go test ./...` now. At one point long ago I think there was an ordering dependency that needed to be satisfied, but I can't see why that wouldn't be a valid way to run the tests now.